plugins/com.google.cloud.tools.eclipse.dataflow.core/src/com/google/cloud/tools/eclipse/dataflow/core/internal/properties/DataflowPropertyTester.java

```diff
@@ -81,12 +80,10 @@ private boolean testIsInDataflowProject(Object receiver) {
 
   private boolean isProjectDataflowProject(IProject project) {
     try {
-      if (project != null && DataflowJavaProjectNature.hasDataflowNature(project)) {
-        return true;
-      }
+      return DataflowJavaProjectNature.hasDataflowNature(project);
```

Comment:
One of these days, we should think about using Nullable where appropriate. In fact, https://github.com/GoogleCloudPlatform/google-cloud-eclipse/issues/2155